libs/common/src/helpers/is-ukef-email.test.ts

```diff
@@ -0,0 +1,22 @@
+import { isUkefEmail } from './is-ukef-email';
+
+describe('isUkefEmail', () => {
+  const invalidEmails = ['', ' ', 'test', 'test@', 'test@example.com', 'test@gov.uk'];
```

Comment:
Can we add some more tests e.g

maker1@ukexportfinance.com
maker1@ukexportfinance.co.uk
admin@ukexportfinance.com
admin@ukexportfinance.co.uk
ukexportfinance@ukexportfinance.gov.uk
maker1@ukef.co.uk

Reply:
Have added.